spec/index.bs

```diff
@@ -1227,29 +1218,27 @@ The {{HTMLScriptElement/src}} setter steps are:
     `script`.</ins>
 1.  <ins>Set [=this=]'s [=src=] content attribute to |value|.</ins>
 
-#### Setting slot values from parser #### {#setting-slot-values-from-parser}
+#### Script children changed steps #### {#script-children-changed-steps}
 
-This document modifies the HTML parser to set the [=script text=] value when the script is created.
+This document modifies the [=children changed steps=] for {{HTMLScriptElement}} as follows:
 
-Modify the [=The text insertion mode=] algorithm as follows:
+1.  <ins>Set [=this=]'s [=HTMLScriptElement/is trusted=] to false.</ins>
 
-<dl class="switch">
-    <dt id="scriptEndTag">An end tag whose tag name is "script"</dt>
-    <dd>
-        <p>...</p>
+1.  <ins>If [=this=]'s [=HTMLScriptElement/changed by trusted sink=] is true, set [=this=]'s [=HTMLScriptElement/is trusted=] to true.</ins>
 
-        <ins><p>Set <var>script</var>'s [=script text=] value to its [=child text content=].</p></ins>
+1.  <ins>Set [=this=]'s [=HTMLScriptElement/changed by trusted sink=] to false.</ins>
 
-        <p>If the <span>active speculative HTML parser</span> is null, then <span>prepare the script
-        element</span> <var>script</var>. This might cause some script to execute, which might cause
-        <span data-x="dom-document-write">new characters to be inserted into the tokenizer</span>, and
-        might cause the tokenizer to output more tokens, resulting in a [=reentrant invocation of the parser=].</p>
+1.  Run the {{HTMLScriptElement|script}} [=HTML element post-connection steps=], given [=this=].
 
-        <p>...</p>
-    </dd>
-</dl>
+Issue: This relies on the children changed steps never being called by the parser.
 
-Issue: The above algorithm doesn't account for the case when the script element's content is changed mid-parse. Implementors should ensure they protect against this case. See [https://github.com/w3c/trusted-types/issues/507](https://github.com/w3c/trusted-types/issues/507).
+This document modifies the [=children changed steps=] for {{SVGScriptElement}} as follows:
+
+1.  Set [=this=]'s [=SVGScriptElement/is trusted=] to false.
+
+1.  Run the {{SVGScriptElement|script}} [=post-connection steps=], given [=this=].
```

Comment:
Does SVG even have this?

Reply:
SVG has very little but we can add a note that we assume it has the same as HTML script has.